_docs-sources/pipelines/hello-world/github-enterprise.md

```diff
@@ -0,0 +1,10 @@
+# GitHub Enterprise
+
+Gruntwork Pipelines uses a set of Gruntwork built re-usable Github Actions. Companies using GitHub Enterprise may need to explicitly allow Actions from Gruntwork to run in their GitHub organization. See the "Allow specified actions and reusable workflows" section in [Allowing select actions and reusable workflows to run](https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#allowing-select-actions-and-reusable-workflows-to-run) to learn more.
+
+You will need to allow the following Actions to run:
+- [pipelines-dispatch](https://github.com/gruntwork-io/pipelines-dispatch)
+- [pipelines-orchestrate](https://github.com/gruntwork-io/pipelines-orchestrate)
+- [pipelines-execute](https://github.com/gruntwork-io/pipelines-execute)
```

Comment:
I don't know that we need to fix it as part of this story (we can just file a bug), but these are public repos, yet trigger the "For subscribers only" alert when you click on them.

<img width="450" alt="image" src="https://github.com/gruntwork-io/docs/assets/4295964/fc5c46cd-8b55-4c63-945a-be0346dd2579">

On a separate note, since those actions are public, we should add more context in the docs about what they accomplish. It's probably enough to add something like this:

# _Pipelines Dispatch GitHub Action_

**_This GitHub Action is part of [Gruntwork Pipelines](https://docs.gruntwork.io/pipelines/overview/), a commercial offering from Gruntwork that enables you to roll out infrastructure changes to prod in a way that meets the needs of your organization._**

_The "Pipelines Dispatch" GitHub Action is designed to dispatch Terragrunt plan/apply/destroy jobs using Gruntwork Pipelines. It facilitates infrastructure change management by triggering the appropriate jobs in the customer's infrastructure-pipelines repository. The Action also adds a link to the logs in pull requests for easy reference._

We should add that to all 3.